main_server/semantic/bmlt_semantic.js

```diff
@@ -981,6 +986,18 @@ BMLTSemantic.prototype.createServiceBodyList = function (
     };
 };
 
+/*******************************************************************************************/
+/**
+ \brief Sort By Key
+ */
+/*******************************************************************************************/
+BMLTSemantic.prototype.sortByKey = function sortByKey(array, key) {
+    return array.sort(function(a, b) {
+        var x = a[key].toLowerCase(); var y = b[key].toLowerCase();
```

Comment:
Can you just use strcasecmp here?

Reply:
I was unaware javascript had such a function, ill check it out